main.go

```diff
@@ -45,9 +45,11 @@ func printVersion() {
 		"go", runtime.Version(),
 		"GOOS", runtime.GOOS,
 		"GOARCH", runtime.GOARCH,
-		"helm-operator", version.Version)
+		"helm-operator", version.Version) // Should not we inform the helm plugin version here? So, should it not be part of SDK impl and no lib?
```

Comment:
// Should not we inform the helm plugin version here? So, should it not be part of SDK impl and no lib?